pkg/iac-providers/arm/v1/types.go

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (C) 2020 Accurics, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+package armv1
+
+// ARMV1 struct implements the IacProvider interface
+type ARMV1 struct {
+	templateParameters map[string]interface{}
```

Comment:
Associate a `multierror` with the struct. Its job would be collecting all the file loading related errors. Please refer to other iac provider implementations.

Reply:
changed